incubator/binding-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/AsyncapiClientCompositeBindingAdapter.java

```diff
@@ -39,13 +40,14 @@ public BindingConfig adapt(
         AsyncapiOptionsConfig options = (AsyncapiOptionsConfig) binding.options;
         AsyncapiConfig asyncapiConfig = options.specs.get(0);
         this.asyncApi = asyncapiConfig.asyncApi;
```

Comment:
```suggestion
        this.asyncapi = asyncapiConfig.asyncapi;
```

Note: for consistency, `asyncapi` is lower case everywhere else.